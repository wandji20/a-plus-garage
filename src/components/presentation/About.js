import React from 'react';

const About = () => (
  <div className="constainer-fluid">
    Home page
    Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    Maecenas iaculis faucibus dui porttitor accumsan. Aenean id sem nunc.
    Quisque at nisi a justo porta pulvinar.
    Morbi a neque ultrices, varius nisi vitae, bibendum velit.
    Quisque sed efficitur augue. Morbi mollis volutpat risus at aliquet.
    Vestibulum tempor lorem vitae nulla blandit, at vehicula purus scelerisque.
    Vivamus sed purus ac ipsum imperdiet consequat.
    Nullam ornare lectus sed libero sodales, ac ornare metus bibendum.
    Praesent nec erat nec velit lobortis lacinia.
    Aenean nec ullamcorper risus, eu ultricies risus.
    Aliquam sollicitudin malesuada purus, ac tempus nunc dignissim in.
    Cras ornare facilisis tellus quis ullamcorper.
    Aenean consectetur risus leo, id interdum odio fringilla vel.
    In rutrum bibendum lorem.
  </div>
);

export default About;
