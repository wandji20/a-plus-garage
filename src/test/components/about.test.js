import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../../redux/store';
import About from '../../components/presentation/About';

it('renders correctly', () => {
  const tree = renderer
    .create(<Provider store={store}><About /></Provider>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
