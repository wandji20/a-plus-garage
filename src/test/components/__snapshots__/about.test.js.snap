// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="container d-flex justify-content-center my-5 py-2"
>
  <h4>
    A Plus Automobile Garage
  </h4>
</div>
`;
